.gitignore

```diff
@@ -15,8 +15,15 @@ _trial_temp*/
 /*.signing.key
 /*.tls.crt
 /*.tls.key
+/*.tls.dh
 /uploads
 /media_store/
+/homeserver*.yaml
+*.db
```

Comment:
hrm, if we're going to have these things, please let's at least anchor them in the root directory:

`/*.db`

Also, please could the entries under "stuff that is likely to exist when you run a server locally" be sorted alphabetically? (and yeah, /media_store ought to be before /uploads)